lib/jsdom/living/node.js

```diff
@@ -15,13 +15,7 @@ exports.clone = (node, document, cloneChildren) => {
   let copy;
   switch (node.nodeType) {
     case NODE_TYPE.DOCUMENT_NODE:
-      // Can't use a simple Document.createImpl because of circular dependency issues :-/
-      copy = document.implementation.createDocument(null, "", null);
-      copy._encoding = node._encoding;
-      copy.contentType = node.contentType;
-      copy._URL = node._URL;
-      copy._origin = node._origin;
-      copy._parsingMode = node._parsingMode;
+      copy = node._cloneDocument();
```

Comment:
IMO, we should keep the note about the circular dependency here. This would make it obvious why the indirection is needed here.